tinker/templates/admin/redirects/home.html

```diff
@@ -175,13 +202,73 @@
                     var url = "{{ url_for('RedirectsView:delete_redirect') }}";
                     $.post(url, redirect_data, function (data) {
                         if (data) {
-                            loadTableForSearch($("#new-redirect-from").val(), 'from_path');
+                            loadTableForSearch($("#new-redirect-from").val(), $('#new-redirect-to').val());
                         }
                     });
                 });
 
+                $(".edit-redirect-btn").click(function () {
```

Comment:
This is like the other comment. Thoughts on simplifying?

Reply:
Same as my comment above too. We need to hide and show each field individually, and this is the only place this particular code appears.